src/cdk/a11y/focus-trap/configurable-focus-trap-factory.ts

```diff
@@ -41,14 +41,22 @@ export class ConfigurableFocusTrapFactory {
   /**
    * Creates a focus-trapped region around the given element.
    * @param element The element around which focus will be trapped.
-   * @param deferCaptureElements Defers the creation of focus-capturing elements to be done
-   *     manually by the user.
+   * @param config The config object. Can also be a boolean representing
+   *   ConfigurableFocusTrapConfig.defer, for backwards compatibility with
+   *   the deprecated FocusTrapFactory create method.
    * @returns The created focus trap instance.
    */
-  create(element: HTMLElement, config: ConfigurableFocusTrapConfig =
+  create(element: HTMLElement, config: ConfigurableFocusTrapConfig | boolean =
```

Comment:
You could alternatively use an overload, which would let us mark the signature as deprecated:
```ts

/** @deprecated Pass a config object instead of the `defer` flag */
create(element: HTMLElement, defer: boolean): ConfigurableFocusTrap;

/** ... */
create(element: HTMLElement, config: ConfigurableFocusTrapConfig = new ConfigurableFocusTrapConfig()) {
  ...
}
```

Reply:
@jelbourn Good point! Done.